aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Host/wwwroot/swagger/ui/abp.swagger.js

```diff
@@ -66,7 +66,11 @@ var abp = abp || {};
         xhr.setRequestHeader('Abp.TenantId', tenantId);
         xhr.setRequestHeader('Content-type', 'application/json');
         addAntiForgeryTokenToXhr(xhr);
-        xhr.send("{" + "usernameOrEmailAddress:'" + usernameOrEmailAddress + "'," + "password:'" + password + "'}");
+        xhr.send(
+            JSON.stringify(
+                { usernameOrEmailAddress: usernameOrEmailAddress, password: password }
```

Comment:
Just passing by, and I am surprised to see no test changes in this PR because of this. I would have thought this would affect existing tests. Maybe there is a test gap. Consider adding tests both for escaped strings and one that ensures that property names are quoted (i.e. the two things mentioned in the original issue https://github.com/aspnetboilerplate/aspnetboilerplate/issues/5081#issue-531158283).